### PR TITLE
feat: add rnl logo to assets

### PR DESCRIPTION
**Pull request by GuillaumeBarra**

### CODE REVIEW

**Comment by ChloeCaronEng** [comment]
[Readability] For the commit name with `fix: deployment`, I cannot tell what you did to fix it, can you edit the commit message to be more descriptive please?

File: src/assets/RNL Logo.svg
Comment (ChloeCaronEng) [inline src/assets/RNL Logo.svg:1]: [Naming] There are a few things with naming conventions to take into account:
- no upper-case letters
- no spaces (use the _ delimiter or - for word combinations)

Comment (ChloeCaronEng) [reply src/assets/RNL Logo.svg:1]: Can these assets be added to a `rnl` folder with the file name as `logo` please?

File: src/pages/events.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { CGLEvent, parseProperties, queryDatabase } from '@/api/notion';
 import Card from '@/components/Card';
 
 import Navbar from '@/components/Navbar';
-import { getPastEvents, getUpcomingEvents, sortEvents } from '@/utils/dateUtils';
```
Comment (ChloeCaronEng) [inline src/pages/events.tsx:5]: [Question] Why was this import removed, was it not being used?

Comment (GuillaumeBarra) [reply src/pages/events.tsx:5]: yes, this is what fixed the deployment (removing unused import), I've renamed the commit to make the changes clearer!

File: .eslintrc.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
       ],
       "extends": [
         "plugin:tailwindcss/recommended",
-        "airbnb-typescript",
+        "airbnb-typescript/base",
```
Comment (ChloeCaronEng) [inline .eslintrc.json]: [Question] Why change the path?

File: .eslintrc.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
         "@next/next/no-img-element": "off", // We currently not using next/image because it isn't supported with SSG mode
         "@typescript-eslint/comma-dangle": "off",
         "no-underscore-dangle": "off",
-        "@typescript-eslint/consistent-type-imports": "error", // Ensure `import type` is used when it's necessary
+        "@typescript-eslint/consistent-type-imports": "off", // Ensure `import type` is used when it's necessary
```
Comment (ChloeCaronEng) [inline .eslintrc.json]: [Question] With the new auto-fix, does this still cause issues?

File: package.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-  "name": "site-natacha",
+  "name": "theodo-uk-meetups",
   "version": "0.1.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint --fix"
```
Comment (ChloeCaronEng) [inline package.json]: Could you add a separate script to do the fix (sometimes you don't want to lint and fix automatically)?

File: src/api/notion.ts [hunk 5]
```diff
@@ -6,38 +6,37 @@ const NOTION_API_KEY = process.env.NOTION_API_KEY ?? '';
 export const notion = new Client({ auth: NOTION_API_KEY });
 
 export const queryDatabase = async () =>
-  await notion.databases.query({
+  notion.databases.query({
```
Comment (ChloeCaronEng) [inline src/api/notion.ts]: [Question] Why remove the await?

File: .eslintrc.json (hunk 3 above)
Comment (GuillaumeBarra) [reply .eslintrc.json]: ah it goes back to the undefined issue in notion.ts if it put it back to error :/

File: .eslintrc.json (hunk 2 above)
Comment (GuillaumeBarra) [reply .eslintrc.json]: was a suggested fix but its not an error anymore, removing it now

File: src/api/notion.ts (hunk 5 above)
Comment (GuillaumeBarra) [reply src/api/notion.ts]: mistake i think

**Review by ChloeCaronEng — APPROVED** [review]
